src/js/BookReader.js

```diff
@@ -640,29 +640,29 @@ BookReader.prototype.drawLeafsOnePage = function() {
   let firstIndexToDraw = indicesToDisplay[0];
   this.updateFirstIndex(firstIndexToDraw);
 
-  if ((0 != firstIndexToDraw) && (1 < this.reduce)) {
-    firstIndexToDraw--;
-    indicesToDisplay.unshift(firstIndexToDraw);
-  }
+  // if zoomed out, also draw prev/next pages
```

Comment:
interesting

Reply:
Yep! I tried removing it, but it made the experience noticeably worse :P